data/mods/More_Survival_Tools/professions.json

```diff
@@ -0,0 +1,110 @@
+[
+    {
+        "type": "profession",
+        "ident": "nomad",
+        "name": "Nomad",
+        "description": "You always prefered the comfort of the open sky, far from the complexities of modern life. Though from the looks of it, things have changed since last time you've been anywhere near civilization.",
+        "points": 4,
+        "skills": [
+            {
+                "level": 4,
+                "name": "survival"
+            },
+            {
+                "level": 3,
+                "name": "fabrication"
+            },
+            {
+                "level": 2,
+                "name": "tailor"
+            },
+            {
+                "level": 3,
+                "name": "gun"
+            },
+            {
+                "level": 2,
+                "name": "archery"
+            },
+            {
+                "level": 3,
+                "name": "cooking"
+            },
+            {
+                "level": 1,
+                "name": "firstaid"
+            }
+        ],
+        "items": {
+            "both": [
+                "armor_farmor",
+                "backpack_leather",
+                "boots_fur",
+                "gloves_fur",
+                "hat_fur",
+                "scarf_fur",
+                "quiver",
+                "sheath",
+                "shortbow",
+                "arrow_wood",
+                "arrow_wood",
+                "copper_knife",
+                "copper_ax",
+                "pot",
+                "waterskin",
+                "fur_rollmat",
+                "flint_steel",
+                "tinderbox"
```

Comment:
Hmm. Hand the hunter the nomad's skills? Would still want to retain the melee and piercing skills, since I decided nomad would only start off competent at archery, combat-wise.